sycl/test/fpga_tests/buffer_location.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
Is test execution disabled by any reason. Could you please add the comment with description if it is the case

Reply:
No, it's a typo. If no objections, I'll do a followup patch to fix that.

Reply:
OK for me to fix in separate patch